Dockerfile

```diff
@@ -0,0 +1,32 @@
+# syntax=docker/dockerfile:experimental
+
+FROM opensuse/tumbleweed:latest
+RUN zypper ref && zypper up -y
```

Comment:
To avoid creating multiple layers and get a smaller image size, I would concatenate these `RUN` commands with `&&`.